test/unit3/test_time.py

```diff
@@ -113,10 +113,9 @@ def test_strftime(self):
         timestamp_to_test = (
             days_to_20020203 * 24 * 60 * 60  # Unix timestamps ignore leap seconds
             + seconds_within_day
-            + time.timezone
         )
         self.assertEqual(
-            time.strftime("%b %d %Y %H:%M:%S", time.localtime(timestamp_to_test)),
+            time.strftime("%b %d %Y %H:%M:%S", time.gmtime(timestamp_to_test)),
```

Comment:
Is there any requirement to use localtime() for this test? If not then using gmtime() instead should resolve the issues.